libs/service-portal/settings/src/components/UserOnboardingModal/UserOnboardingModal.tsx

```diff
@@ -26,6 +31,7 @@ const UserOnboardingModal: ServicePortalModuleComponent = ({ userInfo }) => {
   const [isOpen, setIsOpen] = useState(true)
   const [step, setStep] = useState<OnboardingStep>('intro')
   const [tel, setTel] = useState('')
+  const [buttonLoading, setButtonLoading] = useState<boolean>(false)
```

Comment:
Loading should preferably be handled internally within each step
The loading flag should have a more descriptive name
The loading state should be probably be acquired from the mutation itself instead of a state variable